src/components/biblio/BiblioEntity.js

```diff
@@ -323,18 +497,108 @@ const EntityCreate = () => {
   const disabledEntityList = (taxonSelect === '' || taxonSelect === undefined) ? 'disabled' : '';
   const disabledAddButton = (taxonSelect === '' || taxonSelect === undefined) ? 'disabled' : '';
 
+  if (accessLevel === 'SGD') {
```

Comment:
I still see the code for Entity and Topic Addition on old line 342  new line 606.  Does that mean that STD see something twice ?  (Just trying to understand)

Reply:
I can't tell, is most of the code the same/similar ?  If so, maybe we could make components and reuse them, or have the if (accessLevel === 'SGD')  partially throughout displaying the row.  I'm not sure, I can't tell how similar or different it is.